biogeochem/FatesAllometryMod.F90

```diff
@@ -580,7 +580,10 @@ real(r8) function tree_lai( bl, pft, c_area, nplant, cl, canopy_lai)
     !----------------------------------------------------------------------
 
     if( bl  <  0._r8 .or. pft  ==  0 ) then
-       write(fates_log(),*) 'problem in treelai',bl,pft
```

Comment:
Is that a sufficient warning/error message?  Idea is to then go to that portion of the output?